src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -66,6 +66,12 @@ class Meta:
         verbose_name = _("Objects API configuration")
 
 
+# no catalogus specified, requires both RSIN and domain to be unspecified
+_CATALOGUE_NOT_SET = models.Q(catalogue_domain="", catalogue_rsin="")
```

Comment:
For some reason it never occurred to me to assign these queries to a variable, but it does make things more easy to understand when reading the code :+1:

Reply:
I had the same realisation when writing this line, I'm definitely going to be doing this more often.